client/catapult/extensions/mongo/src/MongoBulkWriter.h

```diff
@@ -31,10 +31,10 @@
 #include "catapult/exceptions.h"
 #include "catapult/types.h"
 #include <boost/asio/io_context.hpp>
-#include <bsoncxx/json.hpp>
-#include <mongocxx/client.hpp>
 #include <mongocxx/config/version.hpp>
 #include <mongocxx/exception/bulk_write_exception.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/client.hpp>
```

Comment:
mongo, bson, mongo looks odd

Reply:
that is because it prioritiezes:
(mongo|bson)/subdir/

and then
(mongo|bson)/

and within them sorts alphabetically...
could split them into separate groups

Reply:
probably would split, but your choice